{{cookiecutter.repository_slug}}/CODE_OF_CONDUCT.md

```diff
@@ -55,7 +55,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at {{cookiecutter.author_email}}. All
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```

Comment:
Will people think about updating this? Or even see it needs to be updated? Wouldn't it be better to default to the email address they provided when creating the project? Just asking for your opinion :slightly_smiling_face:

Reply:
Honestly I'm too much of a nooby to know by experience!!

But yeah I'd definitely put the repo's author here, since he's the only one known at the moment of the list generation.
And people can always edit that later if necessary.

You give me too much credit, I think I just pasted the latest code of conduct and overlooked this :D
Mayyyy-be I had a thought about privacy but that's bs.